portfolio/src/main/webapp/comments.js

```diff
@@ -53,4 +71,15 @@ function addAllCommentsToDOM(comments){
         addCommentToDOM(comments[i]);
         console.log(comments[i]);
     }
+}
+
+function removeAllCommentsFromDOM(){
+    var commentCardHolders = document.getElementsByClassName("card-holder-small");
+    console.log(commentCardHolders.length + " cards on the page");
+    var noCards = commentCardHolders.length;
+    for(var i=noCards-1; i>=0; i--){
```

Comment:
Wondering if we could simply remove all children from "main-comments" so we don't have to individually remove them? Is there such a DOM function?

Reply:
I restructured a little the main element, and then I set innerHTML to an empty string to remove all the children